sway-core/src/transform/to_parsed_lang/convert_parse_tree.rs

```diff
@@ -279,6 +280,18 @@ fn item_use_to_use_statements(
         &mut ret,
         item_span,
     );
+
+    // Check that all use statements have a call_path
+    // This is not the case for `use foo;`, which is currently not supported
```

Comment:
I don't think those types of imports were ever part of the spec, so I don't think we have an issue tracking this.
Can you create one and mention it here?

Reply:
Will do.